qa/pull-tester/rpc-tests.py

```diff
@@ -313,10 +318,6 @@ def run_tests(test_list, src_dir, build_dir, exeext, jobs=1, enable_coverage=Fal
     else:
         coverage = None
 
-    if len(test_list) > 1 and jobs > 1:
-        # Populate cache
-        subprocess.check_output([tests_dir + 'create_cache.py'] + flags)
```

Comment:
Why did this need to be moved before the `enable_coverage` line? AFAICT the only change to the moved code is to disable it if `SHARD_SCRIPT` is set, so in the context of this PR I don't understand why that change couldn't just be made here without the move.